app.js

```diff
@@ -0,0 +1,23 @@
+// Fetching books on main html page //
+let allBooks = document.querySelector(".books");
+window.addEventListener("DOMContentLoaded", getBooks);
+function getBooks() {
+  fetch("https://bookshop-api.mirkwood.dev/books/")
+    .then(function (re) {
+      return re.json();
+    })
+    .then(function (data) {
+      let bookDiv = "";
```

Comment:
As I said in a earlier PR, please create a new function outside getBooks(), for example displayBooks(books) in which you will add the code of creating the book DOM elements and adding them to the page